tests/.pest/snapshots/Countries/SyriaTest/it_can_translate_syria_holidays_into_arabic.snap

```diff
@@ -0,0 +1,58 @@
+[
+    {
+        "name": "New Year's Day",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Mother's Day",
+        "date": "2024-03-21"
+    },
+    {
+        "name": "Teacher's Day",
+        "date": "2024-03-21"
+    },
+    {
+        "name": "\u0639\u064a\u062f \u0627\u0644\u0641\u0635\u062d \u0627\u0644\u063a\u0631\u0628\u064a",
+        "date": "2024-03-31"
+    },
```

Comment:
A combination of languages here

Reply:
I have fix this issue the problem was in the ('s)